src/java/test/org/apache/zookeeper/test/QuorumMajorityTest.java

```diff
@@ -36,7 +38,30 @@
     /***************************************************************/
     @Test
     public void testMajQuorums() throws Throwable {
-       
+        LOG.info("Verify QuorumPeer#electionTimeTaken jmx bean attribute");
+
+        ArrayList<QuorumPeer> peers = getPeerList();
+        for (int i = 1; i <= peers.size(); i++) {
+            QuorumPeer qp = peers.get(i - 1);
+            Long electionTimeTaken = -1L;
+            String bean = "";
+            if (qp.getPeerState() == ServerState.FOLLOWING) {
+                bean = CommonNames.DOMAIN + ":name0=ReplicatedServer_id" + i
+                        + ",name1=replica." + i + ",name2=Follower";
```

Comment:
Thanks @rgs1, I've have modified the test code as per your suggestion. Please take another look at the new commits.